src/data/exercises.json

```diff
@@ -0,0 +1,1040 @@
+          "answers": [
+            {
+              "id": 331,
+              "content_html": "0",
+              "correctness": "0.0",
+              "feedback_html": "Do you think external forces include those forces also which are originating from inside the system? Can you differentiate between external and internal forces?"
+            },
+            {
+              "id": 330,
+              "content_html": "1",
+              "correctness": "1.0",
+              "feedback_html": "Correct! External force is a force that acts on the object of interest rather than being acted on by the object. The external forces do not include forces originatingf rom inside the system."
+            }
+          ],
+          "hints": [],
+          "formats": [
+            "multiple-choice",
+            "free-response"
+          ],
+          "combo_choices": []
+        }
+      ]
+    },
+    {
+      "uid": "16@1",
+      "number": 16,
+      "version": 1,
+      "published_at": "2015-03-10T19:49:18.057Z",
+      "editors": [],
+      "authors": [
+        {
+          "user_id": 1
+        }
+      ],
+      "copyright_holders": [
+        {
+          "user_id": 2
+        }
+      ],
+      "derived_from": [],
+      "attachments": [],
+      "tags": [
+        "k12phys-ch04-s01-lo01",
+        "k12phys-ch04-ex016",
+        "test-prep-short-answer",
+        "inbook-yes",
+        "dok2",
+        "time-short",
+        "display-free-response",
+        "display-rec-sensitive"
+      ],
+      "stimulus_html": "",
+      "questions": [
+        {
+          "id": 94,
+          "stimulus_html": "",
+          "stem_html": "By convention, which sign is assigned to an object moving downward?",
+          "answers": [
+            {
+              "id": 335,
+              "content_html": "No sign is assigned",
+              "correctness": "0.0",
+              "feedback_html": "Can a vector quantity have no sign for its direction?"
+            },
+            {
+              "id": 334,
+              "content_html": "Can be either positive or negative",
+              "correctness": "0.0",
+              "feedback_html": "Do you think it is true as per the convention?"
+            },
+            {
+              "id": 333,
+              "content_html": "Negative sign",
+              "correctness": "1.0",
+              "feedback_html": "Correct! Negative sign is assigned to an object moving downward."
+            },
+            {
+              "id": 332,
+              "content_html": "Positive sign",
+              "correctness": "0.0",
+              "feedback_html": "Are you sure that positive sign is assigned to an object moving downward? Try and think if this is true as per the convention."
+            }
+          ],
+          "hints": [],
+          "formats": [
+            "multiple-choice",
+            "free-response"
+          ],
+          "combo_choices": []
+        }
+      ]
+    }
+  ]
+}
```

Comment:
Can you run this through `http://jsbeautifier.org` (with 2 spaces option)?

Also, for largely [historical reasons](http://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file), all files in webview should end in a new-line (ie, have a blank last line)